templates/client-daemonset.yaml

```diff
@@ -126,8 +133,20 @@ spec:
                 -advertise="${ADVERTISE_IP}" \
                 -bind=0.0.0.0 \
                 -client=0.0.0.0 \
+                {{- if .Values.global.tls.enabled }}
```

Comment:
Yeah, I think it would be nice. My only reservation against it is that it currently requires a manual restart. So disabling/enabling through the Helm chart on an existing cluster won't do anything.

Reply:
Oh good catch. Okay.